src/StanfordSentimentTreebank.jl

```diff
@@ -0,0 +1,31 @@
+"""StanfordSentimentTreebank Type
+having dict and senti fields for
+phrases and corresponding sentiments respectively"""
+struct StanfordSentimentTreebank{S}
+    dict :: Array{S, 2}
+    senti :: Array{S, 2}
+end
+
+function StanfordSentimentTreebank()
+    dictionary = readdlm(joinpath(datadep"Stanford Sentiment Treebank", "dictionary.txt"), '|')[2:end, :] #dictionay.txt files is import here from dataset directory
+    sentiments = readdlm(joinpath(datadep"Stanford Sentiment Treebank", "sentiment_labels.txt"), '|')[2:end, :] #corresponding sentiments files from dataset
+    return StanfordSentimentTreebank(dictionary, sentiments)
+end
```

Comment:
We do not want to be loading large files in the constructor.
That is what the `load` function is for.

Reply:
I will fix this soon